templates/case-study/grundium-ubuntu-pro-for-devices.html

```diff
@@ -0,0 +1,202 @@
+{% from "macros/_macros-quote.jinja" import quote_wrapper %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+{% from "macros/_macros-cta.jinja" import cta %}
+
+{% block title %}Grundium secures critical systems with Ubuntu Pro for Devices{% endblock %}
+
+{% block meta_description %}
+  As a medical device manufacturer, Grundium must meet stringent security requirements. Ubuntu Pro for Devices allows them to secure their digital microscopes.
+{% endblock %}
+
+{% block meta_company_name %}Grundium{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1wM1vo24SFowPaAnxWV9wfU-w6PVDq7RczM_5ZOb8OZM/edit
+{% endblock %}
+
+{% block content %}
+  {%- call(slot) hero(title="Canonical deploys infrastructure solutions and managed IT services for critical space mission operations",
+                      layout="25/75",
+                      is_split_on_medium="true") -%}
+    {% if slot == "signpost_image" %}
+      {{ image(url="https://assets.ubuntu.com/v1/547c1ef8-Grundium-logo-black 1.png",
+            alt="",
+            width="264",
+            height="53",
+            hi_def=True,
+            loading="auto") | safe
+      }}
+    {% endif %}
+  {%- endcall -%}
+
+  {%- call(slot) text_list(title="About Grundium") -%}
+    {%- if slot == 'list_item_content_1' -%}
+      Grundium is a medical device manufacturer that specialises in digital microscopy. Their range of microscope scanners are known as Ocus® Scanners.
+    {%- endif -%}
+    {%- if slot == 'list_item_content_2' -%}
+      These high-resolution digital microscopes allow pathologists to collaborate remotely, thus avoiding the need for physical samples to be sent to different locations.
+    {%- endif -%}
+    {%- if slot == 'list_item_content_3' -%}
+      As a company that innovates in the medical field, Grundium must meet stringent security requirements around patient data and confidentiality.
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) highlights(num_items=3) -%}
+    {%- if slot == 'highlights_item_content_1' -%}
+      Grundium’s digital microscopes are used across the medical industry. To comply with strict data and security frameworks, they chose Ubuntu Pro for Devices as a straightforward way to secure their devices.
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_2' -%}
+      Canonical maintains the essential open source software components and offers built-in compliance with the major frameworks, allowing the company to innovate with confidence across the medical industry.
+    {%- endif -%}
+    {%- if slot == 'highlights_item_content_3' -%}
+      Ubuntu Pro for Devices offers regular patching and over-the-air updates, ensuring that Grundium’s scanners remain secure and stable, wherever they are located.
+    {%- endif -%}
+  {%- endcall %}
+
+  {%- call(slot) image_wrapper(caption="An Ocus® Scanner in a laboratory, connected remotely to a laptop.") -%}
```

Comment:
I doubt if this will work here, go through the code and wherever possible replace &reg;

Reply:
I've updated all the instances of ® to `<sup>&reg;</sup>`, but the `sup` tags cannot be added to image slots. I've replaced them with `&reg;` as without the tags as they still look the same.